Makefile

```diff
@@ -1,5 +1,5 @@
 DUCKDB_REPO=https://github.com/duckdb/duckdb.git
-DUCKDB_BRANCH=main
+DUCKDB_BRANCH=c29c67bb971362cd1e9143305acffebb1bc9bd63
```

Comment:
@srikrishnak can you please submit a patch that renames this DUCKDB_REF?

It isn't a branch anymore.

Reply:
How I did in go-duckdb:

https://github.com/sundeck-io/go-duckdb/blob/main/Makefile#L23

Reply:
took care of rename to DUCKDB_REF in PR https://github.com/sundeck-io/duckdb-go-extensions/pull/8